cirq-core/cirq/transformers/heuristic_decompositions/gate_tabulation_math_utils.py

```diff
@@ -58,7 +59,9 @@ def random_qubit_unitary(
         rng: Random number generator to be used in sampling. Default is
             numpy.random.
     """
-    real_rng: np.random.RandomState = np.random if rng is None else rng
+    real_rng: np.random.RandomState = (
+        random_state.parse_random_state(np.random) if rng is None else rng
+    )
```

Comment:
Please replace with a simple call to `parse_random_state()`

Reply:
Done.